### DISCUSSION TITLE
RainbowKit <> WAGMI v1.0 + Viem Intergration

### DISCUSSION THREAD

**Discussion by Seranged** [rainbow-me/rainbowkit#1221] [Ideas]
I'm sure you guys have had this planned in your roadmap but WAGMI V1.0 got released today which removes Ethers.js and implements Viem. 

Just created an [Feat] idea discussion so people and everyone else see and can upvote it!

Cheers guys thank you for all your work!

**Comment by zkSoju** [comment]
Just waiting on the release, I think the PR is already in.

**Comment by Seranged** [comment]
PR came in just after I posted this! They're fast 🫡
